app/models/pipeline_run.rb

```diff
@@ -726,6 +794,17 @@ def check_and_log_long_run
     end
   end
 
+  def load_chunk_stats
```

Comment:
This is (1) rather costly and (2) only a perf/cost optimization.   It may benefit from being run not more often than every 10 minutes.  Particularly if there are tons of jobs.  Perhaps it would be worthwhile doing a performance experiment with lots of jobs where the actual gsnap and rapsearch ssh is stubbed out to just "sleep 10 minutes and copy known outputs to result folder" which would give you a chance to measure the impact of these result pulls on job scheduling and other latencies.